### PR TITLE
Tools: Topology2: Build nocodec topology for PTL with 96 kHz SSP

### PR DESCRIPTION
**Pull request by singalsu**
This patch adds possibility to build with cavs-nocodec.conf topology
in passthrough mode with other rate than 48 kHz. The topologies built
are:

- sof-ptl-nocodec-dmic-4ch-96k-ssp0-96k.tplg
- sof-ptl-nocodec-fpga-dmic-4ch-96k-ssp0-96k.tplg

Signed-off-by: Seppo Ingalsuo <seppo.ingalsuo@linux.intel.com>

### CODE REVIEW

File: tools/topology/topology2/development/tplg-targets.cmake [hunk 1]
```diff
@@ -63,6 +63,11 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-ptl-nocodec.bin"
 "cavs-nocodec\;sof-ptl-nocodec-dmic-4ch-96k\;PLATFORM=ptl,NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,\
 DMIC0_RATE=96000,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-ptl-nocodec-dmic-4ch-96k.bin"
 
+# SSP topology for PTL with 96 kHz DMIC and 96 kHz SSP0
+"cavs-nocodec\;sof-ptl-nocodec-dmic-4ch-96k-ssp0-96k\;PLATFORM=ptl,NUM_DMICS=4,\
+PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,DMIC0_RATE=96000,SSP0_RATE=96000,\
+PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-ptl-nocodec-dmic-4ch-96k-ssp0-96k.bin"
```
Comment (singalsu) [inline tools/topology/topology2/development/tplg-targets.cmake:69]: Should have here PASSTHROUGH=true

Comment (singalsu) [reply tools/topology/topology2/development/tplg-targets.cmake:69]: Fixed!

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
@kv2019i @ranj063 pls review
@singalsu conflicts

**Comment by singalsu** [comment]
This is waiting for #9317. I might also add 192 kHz if I'm able to test it.

**Comment by lgirdwood** [comment]
> This is waiting for #9317. I might also add 192 kHz if I'm able to test it.

@singalsu now merged.

**Comment by singalsu** [comment]
Replaced by #9619.

**Pull request closed without merging**